frontend/public/components/routes.tsx

```diff
@@ -295,6 +268,9 @@ const RouteDetails: React.SFC<RoutesDetailsProps> = ({obj: route}) => <React.Fra
       </div>
     </div>
   </div>
+  {(route.status && route.status.ingress) && <div className="co-m-pane__body">
```

Comment:
Maybe something like

```jsx
{_.isEmpty(route.status.ingress)
  ? <div className="cos-status-box">
      <div className="text-center">No Route Status</div>
    </div>
  : <IngressDetails ingresses={route.status.ingress} />}
```

nit: please put a space before `/` in self-closing tags throughout for consistency with existing code